python/src/aiconfig/default_parsers/openai.py

```diff
@@ -106,6 +106,17 @@ async def serialize(
                     messsage["content"] if role == "user" else PromptInput(**messsage)
                 )
 
+                assistant_output = []
+                if assistant_response is not None:
+                    assistant_content = assistant_response.pop("content", "")
+                    assistant_output = [
+                        ExecuteResult(
+                            output_type="execute_result",
+                            execution_count=None,
+                            data=assistant_content,
+                            metadata={**assistant_response},
```

Comment:
It's important to ensure the output produced by TS and Python SDKs is the same. This logic seems to produce different things in the metadata between the SDKs. If that is intentional, it should be explicitly documented

Reply:
Thanks for the callout, will update so that they're both `metadata: {rawResponse: assistantResponse}`

Also yea I know this isn't handling function calling yet, but I'll update in future diff at the end of stack. Sarmad and I discussed offline but sharing so others can see, we want metadata eventually to be something like this:

```
metadata: {rawResponse: response, ..._.omit(response, ["contents"])}
```
while using `function_call` kind in the output.data: https://github.com/lastmile-ai/aiconfig/pull/628